Client/src/Pages/Auth/Login.jsx

```diff
@@ -126,18 +156,26 @@ LandingPage.propTypes = {
 const StepOne = ({ form, errors, onSubmit, onChange, onBack }) => {
 	const theme = useTheme();
 	const inputRef = useRef(null);
-
+	const navigate = useNavigate();
 	useEffect(() => {
 		if (inputRef.current) {
 			inputRef.current.focus();
 		}
 	}, []);
 
+	const handleNavigate = () => {
+		if (form.email !== "" && !errors.email) {
+			sessionStorage.setItem("email", form.email);
```

Comment:
What does this have to do with radio button styling?

This issue and PR is for radio buttons only 🤔

What are you trying to resolve here?

If you want to work on a separate issue please open a separate PR for it